content/using/operations/using-bridge.md

```diff
@@ -42,7 +42,12 @@ If you were given points by Tlon you likely already fully own them. But if someo
 
 After you access your Ethereum address, if a point was sent to that address, you'll come to a page that has an `Incoming Transfers` header, under which is a graphic. Click the `Details ->` link under that graphic.
 
-Now you'll be on the management page of your point. The transfer isn't completed yet, so click `Accept incoming transfer`. Then check both boxes and and click their associated `Sign Transaction` and `Send Transaction` buttons.
+Now you'll be on the management page of your point. The transfer isn't completed
+yet, so click `Accept incoming transfer`. If you are transferring to yourself
+and do not wish to [breach](@/docs/glossary/breach.md), check the box labeled `Retain
+proxies and key configuration, in case of transferring to self`. Otherwise leave
+the box unchecked, ensuring that your ship will be breached upon transfer. Then
```

Comment:
Might be better to rephrase as "will not retain any data from the previous owner", to emphasize the purpose of the behavior here. (Or, "breach and not retain ..." or w/e.)

Reply:
https://github.com/urbit/urbit.org/pull/505/commits/a8a0a27a7332a19b79ef04c27f695cba8c0dd888